packages/lib-classifier/src/store/WorkflowStore/WorkflowStore.js

```diff
@@ -61,7 +61,8 @@ const WorkflowStore = types
         self.resources.put(workflow)
         if (subjectSetID) {
           const selectedWorkflow = self.resources.get(id)
-          selectedWorkflow.selectSubjectSet(subjectSetID)
+          // wait for the subject set to load before activating the workflow
+          const subjectSet = yield selectedWorkflow.selectSubjectSet(subjectSetID)
```

Comment:
Same question about error catching here, I'm not sure if/where an error would be caught for this part of the loading.

Reply:
I'm not sure either. The classifier renders with an empty subject viewer, so the error is being caught somewhere. Here's the classifier when I edit the URL to include an invalid subject set ID.

![Screenshot of the classifier with an empty subject set viewer and the dev tools open to show MobX State Tree invalid reference errors for the subject sets store.](https://user-images.githubusercontent.com/59547/93577369-dce93080-f993-11ea-83c1-322b86d22c1d.png)